float64     estimated_angle
float64     measured_angle
float64[]   estimated_position
float64[]   measured_position